docs/history.rst

```diff
@@ -14,7 +14,7 @@ around many of the open source repos (and some of the Sentry dependencies)
 you'll see a common thread: StarCraft 2. Originally we started with the
 name Sentry representing a Protoss unit that had a shield, which was
 fairly fitting as Sentry helps shield you from problems, but doesn't
-prevent them, just like the unit. When we named the client we went with
+prevent them, just like the unit. When we named the SDK we went with
```

Comment:
seems kind of awkward to say "named the SDK", maybe "when we named the first SDK" or something?